server.js

```diff
@@ -89,6 +96,7 @@ statics.on('add', (filePath) => {
 
 // Initializes the server and mounts it in the generated build directory.
 http.createServer((req, res) => {
+  if (wellknown(req, res)) return
```

Comment:
This server is only used for development so I'm not sure if it's worth adding. In production the site is served by NGINX.

Reply:
Then in production it would be good to add `/security.txt` -> `/.well-known/security.txt` redirect as recommended by the spec and employed by a couple of bigger sites on the Web.

Reply:
Yes, it should be added here: https://github.com/nodejs/build/blob/master/setup/www/resources/config/nodejs.org

Reply:
@lpinca `security.txt` should be in the repo, file would be served in production like our static files.

We just need to add 
```
location /security.txt {
    alias /home/www/nodejs/.well-known/security.txt;
    default_type text/plain;
}
```

in the nginx config.

Reply:
Yes exactly that's what I was suggesting.

Reply:
We don't need the build step and the addition to `server.js` though as we don't care in development, I guess.

Reply:
Same build script is used in production and we need a way to place the file in the `build` directory.

Reply:
> we need a way to place the file in the build directory.
You are right.

Reply:
@fhemberger on second thought, why should it be in `build`? `robot.txt` for example is in `static`.

```
location /robots.txt {
    alias /home/www/nodejs/static/robots.txt;
    default_type text/plain;
}
```

Reply:
Wouldn't mind it at all. Keeps things simple. 👍

Reply:
OK, so the bottom line is to put `security.txt` into the `static` folder and handle `/security.txt` and `/.well-known/security.txt` in NGINX config and get rid of everything else in `server.js` and `build.js`?

Reply:
That works for me.

Reply:
Thanks for all the feedback, I will make a change and submit a second PR against nodejs/build.

Reply:
Thanks for all the feedback, I will make a change and submit a second PR against nodejs/build.